CodeforAustralia/council-emissions-calculator
# bug fix for issue #227

fixes #227 

## Overview of the Pull Request:
* the bug reported in #227 was due to outdated SSL libraries in the version of webpack used in nextJS v11. this PR upgrades nextJS to v12
* also updated the following packages to address some security issues:
  * svgr
  * reactJS to v18

## link to Trello card or Github issue
#227 

## How Has This Been Tested?
1. install and use the latest lts version using nvm:
  ```
  nvm install v18.12.1
  nvm use
  ```
2. remove `node_modules/` folder, and run `npm install`
3. run `npm run dev`
4. Should no longer see error message reported in #227 

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!
